docs/rules/no-bitwise-trunc.md

```diff
@@ -0,0 +1,23 @@
+# Enforce the use of `Math.trunc` instead of `| 0`
+
+Enforces a convention of using [`Math.trunc()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc) instead of `| 0` (bitwise OR with 0) for clarity and more reliable results.
+
+The `| 0` hack helps truncate numbers but is not clear and does not work in some cases.
```

Comment:
Can you linkify `some cases` to more info about that?

Reply:
Sure! Done in the next commit.